pkg/config/config.go

```diff
@@ -369,6 +369,16 @@ func loadProxyFromEnv() {
 		return os.Getenv(strings.ToLower(key))
 	}
 
+	// getEnvDDPrecedence pulls the value of the `DD_`-prefixed env var first, and
+	// if not found or empty, the value of the env var without the prefix (case-insensitive)
+	getEnvDDPrecedence := func(key string) string {
+		value := os.Getenv("DD_" + key)
```

Comment:
don't we want to be case insensitive on this `key` as well?

Reply:
`viper` is case-sensitive on the env variables. Since this env var is meant to be pulled like any other env var pulled by `viper` I think it's better to be case-sensitive for consistency.

I can add a comment to clarify this if you think it's worth it